packages/dds/tree/src/core/rebase/changeRebaser.ts

```diff
@@ -97,6 +101,11 @@ export interface TaggedChange<TChangeset> {
 	 * Considered false if undefined.
 	 */
 	readonly isInverse?: boolean;
```

Comment:
The `isInverse` field original meant essentially the same thing as `isRollback`. I don't think we're using `isInverse` anywhere, and it seems reasonable to remove it as part of this PR.